posts/inside-rust/2019-11-12-goverance-wg-cfp.md

```diff
@@ -0,0 +1,64 @@
+# Governance WG Call For Participation
+
+Hello everyone, the governance working group has been working a few efforts, but
+we haven't made as much progress as we would have liked over the past few
+months, so we are going to try out a new process and meeting agenda aimed at
+trying to get more work done in the time we know we have.
+
+## New Meeting Structure
+
+Previously we have been doing most of our work async while using our bi-weekly
+meeting call to triage tasks. The problem we ran into with this is that a lot of
+the members also heavy contributors to the Rust project as a whole, and it can
+be can hard for some people to schedule time write an initial draft.
+
+To address this instead of triaging during the meeting and working on tasks in
+async, we're going to focused topic based meetings, and use the time between
+meetings, to publish posts like this and to research and prepare for the next
+topic. To help do this we're going to extend our current meeting duration from
+30 minutes to an hour. (The meeting will still be every two weeks.)
+
+The current goals are to documenting the de-facto governance structure, provide
+the result as a RFC and then if merged provide a version on
+<forge.rust-lang.org> so that it has greater visbility. We also want to try
+schedule people involved in Rust and other governance structures to come and
+talk about their experiences.
+
+For deciding what topics, we're going to have rotating person for each meeting
+who takes the lead for the next meeting. It's that person's responsibility to
+decide the topic and to prepare a blog post similar to this informing people of
+the topic and how they might best prepare.
+
+We also hope that having a focused topic will reduce any barrier of expected
+knowledge in order to participate and contribute. With that said let's talk
+about the topic for next meeting.
+
+## Reviewing & Examining Previous Governance RFCs
+
+Our first topic for new meeting is going to be to read [RFC 1068] aka the
```

Comment:
```suggestion
Our first topic for new meeting is going to be to read [RFC 1068], the
```

Or: "the original Rust Governance RFC (RFC 1068)", which would probably be less confusing than commas.